src/library_wasmfs.js

```diff
@@ -161,7 +161,13 @@ var WasmfsLibrary = {
     wasmFS$JSMemoryFiles[index] = null;
     // Add the index to the free list.
     wasmFS$JSMemoryFreeList.push(index);
-  }
+  },
+  _emscripten_get_preloaded_file_size: function(index) {
+    return wasmFS$preloadedFiles[index].fileData.length;
+  },
+  _emscripten_copy_preloaded_file_data: function(index, buffer) {
```

Comment:
How about we rename all these methods to be `_wasmfs_foo_bar`.. which would make them shorter and more usefully named?  (Can be a followup PR obviously).

As it stands I can't tell form the function names that they relate to wasmfs.